docs/tutorials/first_app/blinky_compiler.md

```diff
@@ -1,70 +1,81 @@
+- Plug your Mbed board into your computer, and open its USB device folder.
+- Double click on the `MBED.HTM` file. (This adds your Mbed platform to the Online Compiler.)
 
-You're taken to the online IDE, and the **Import Program** dialog box opens:
+If you do not have an Mbed board, go to [os.mbed.com/platforms](http://os.mbed.com/platforms), select a board and click the “Add to your Mbed Compiler” button.
+<span class="images">![](https://sarahmarshy.github.io/img/add-to-compiler.png)
+</span>
 
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/import_dialog.png)<span>Importing Blinky</span></span>
+### Code
 
-The import mechanism offers a default name, but you're free to change it. When you're done, click **Import**.
+#### Import 
 
-### Viewing Blinky
+Visit the Mbed OS [blinky example repository](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-blinky/), and click the "Import into Compiler" button.
+ 
+<span class="images">![](https://sarahmarshy.github.io/img/import-compiler.png)
+</span>
 
-The imported Blinky has two interesting parts:
+#### Compile
 
-- `main.cpp`, where the Blinky-specific code is. You can double-click the file in the navigation pane on the left to view the code.
-- `mbed-os`, where the Arm Mbed OS codebase is.
+Click on the "Compile" button. Your browser downloads the program as an executable file.
 
-Later we'll compile the code; this will take both of these parts and create a single application file from them.
+<span class="images">![](https://sarahmarshy.github.io/img/compile.png)
+</span>
 
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/maincpp.png)<span>Viewing the code in main.cpp</span></span>
+#### Program
 
-### Selecting a target board
+Open the folder where the executable file was downloaded, and then click and drag (or copy and paste) the file to your Mbed board's USB device folder.
 
-The Mbed Online Compiler can build your application to match any Arm Mbed Enabled board. However, you have to select the target board before compiling.
+Once you have flashed the file to the board, press the board's "reset" button. The LED blinks.
 
-#### Adding a board to your list
+### Debug
 
-To add a board to your list, go to <a href="https://os.mbed.com/platforms/" target="_blank">the board's page on `os.mbed.com`</a>, and click the **`Add to your Mbed Compiler`** button:
+#### Desktop IDE
 
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/add_board.png)<span>Adding a board to the Mbed Online Compiler's board list</span></span>
+To debug using a desktop IDE such as Keil uVision, IAR or Eclipse, click the `Export` button under `Program Details`. Select your export platform and IDE, and click `Export`. Your browser downloads a `.zip` file with the project files.
 
-#### Selecting a board
+<span class="images">![](https://sarahmarshy.github.io/img/export.png)
```

Comment:
This image is out of date. Please update it and upload it to the "images" folder at a higher level of the Handbook.

Reply:
@yennster can you please update the image?
@AnotherButler where would you like us to upload the image? should we create a `img` folder or is there a master image folder we should upload to?

Reply:
As mentioned previously, there's an "images" folder at a high level in our documentation where we put all images: https://github.com/ARMmbed/Handbook/tree/new_engine/docs/images

Reply:
@AnotherButler @BlackstoneEngineering Sorry, I just saw this note today. Do you still need those images uploaded?

Reply:
@yennster Yes, please. If you could upload these at your convenience, that would be great. Thanks for checking.

Reply:
@AnotherButler will do later today!